modules/manage/pages/iceberg/iceberg-schema-changes-and-migration-guide.adoc

```diff
@@ -0,0 +1,145 @@
+= Iceberg Schema Changes and Migration Guide for Redpanda v25.3
+:description: Information about breaking schema changes for Iceberg topics in Redpanda v25.3, and actions to take when upgrading.
+:page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
+
+Redpanda v25.3 introduces changes that break table compatibility for Iceberg topics. Before you upgrade to v25.3, you must take specific actions to ensure that your Iceberg topics and their associated tables continue to function correctly.
+
+== Breaking changes
+
+The following table lists the schema changes introduced in Redpanda v25.3.
+
+|===
+| Field | Iceberg type translation before v25.3 | Iceberg type translation starting in v25.3 | Impact
+
+| `redpanda.timestamp` column 
+| `timestamp` type 
+| `timestamptz` (timestamp with time zone) type
+| Affects all tables created by Iceberg topics, including DLQ tables.
+
+| `redpanda.headers.key` column 
+| `binary` type 
+| `string` type 
+| Affects all tables created by Iceberg topics, including DLQ tables.
+
+| Avro optionals (two-field union of `[null, <FIELD>]`)
+| Single-field struct type
+| Optional `FIELD`
+| Affects tables created by Iceberg topics that use Avro optionals.
+
+| Avro union 
+| 
```

Comment:
SME review: please confirm the previous <25.3 behavior, and can we please add an example to illustrate?